### ISSUE TITLE
Anaconda navigator not opening behind proxy

### ISSUE THREAD

**Issue by vermaprima** [ContinuumIO/anaconda-issues#12136]
I downloaded Anaconda yesterday on my office PC. It got installed perfectly. However, the navigator is not opening, probably because my PC is behind a proxy. when I double click the navigator icon, it just gets stuck at "Loading application". When I use Anaconda prompt to open the navigator, I get the following in return:

![Untitled1](https://user-images.githubusercontent.com/73879043/97967678-933d9380-1de3-11eb-9abf-c9a2a89ef037.jpg)

Can somebody please help me to open the navigator, please? Basically, if I can get where and how do I update the proxy settings (address, and the username/ password), it might solve the problem. But I have no clue on how/ where to do that.

**Comment by csoja** [comment]
There is a page in the documentation that might be of help: https://docs.anaconda.com/anaconda/user-guide/tasks/proxy/  However, you may need to talk to your IT organization for help on your particular proxy setup.

**Comment by vermaprima** [comment]
Yes, I have updated the .condarc file. However, the problem still persists. My company IT is also not able to solve this.
